Documentation/dev/libvirt-howto.md

```diff
@@ -7,16 +7,44 @@ for operator development.
 ### 1. One-time setup
 It's expected that you will create and destroy clusters often in the course of development. These steps only need to be run once (or once per RHCOS update).
 
-#### 1.1 Pick a name and ip range
+#### 1.1 Install Dependencies and Enable Libvirt
+First install some dependencies needed to complete these instructions and then
+enable the libvirt daemon.
+
+```sh
+sudo dnf copr enable vbatts/bazel
+sudo dnf install bazel golang-bin gcc-c++ libvirt-daemon
+sudo systemctl start libvirtd
+sudo systemctl enable libvirtd
+```
+
+Check to see if a default storage pool has been defined in Libvirt by running
+`sudo virsh pool-list`.  If it does not exist, create it:
```

Comment:
I dunno how portable this is, but on my RHEL 7.5 CSB system I don't need root:

```console
$ virsh -c qemu:///system pool-list
 Name                 State      Autostart 
-------------------------------------------
 default              active     yes       

```

Reply:
A later step in libvirt-howto shows how to enable non-root access.  Perhaps you've already done that?  At this point in the instructions though, I was assuming it hadn't been done.

Reply:
> A later step in libvirt-howto shows how to enable non-root access. Perhaps you've already done that?

I don't remember having done that, and I don't have the polkit rule mentioned there:

```console
$ sudo ls -l /etc/polkit-1/rules.d/
total 8
-rw-r--r--. 1 root root 974 Jan 26  2014 49-polkit-pkla-compat.rules
-rw-r--r--. 1 root root 326 Apr 29  2013 50-default.rules
```

It would be nice to have a vanilla RHEL system to test on.

Reply:
OK, thanks.  I'll adjust it and then try all this again some other time when I have a fresh system again.

Reply:
I'm also going to move this section down by the other libvirt config related steps